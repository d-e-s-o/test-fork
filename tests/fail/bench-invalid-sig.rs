// Copyright (C) 2025 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

/// Invalid signature for `#[bench]` function.
#[test_fork::bench]
fn invalid_sig() {}

fn main() {}
