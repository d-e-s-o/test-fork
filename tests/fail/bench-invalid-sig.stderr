error: benchmark function has unexpected signature (expected single `&mut Bencher` argument)
 --> tests/fail/bench-invalid-sig.rs:6:1
  |
6 | fn invalid_sig() {}
  | ^^^^^^^^^^^^^^^^
